src/core/chatController.js

```diff
@@ -49,50 +52,75 @@ class ChatController {
     this._sendInternalLogToServer(this.logger.info("Subscribed successfully to event:", eventName));
   }
 
-  handleRequestSuccess(metadata) {
+  handleRequestSuccess(metadata, method, startTime, contentType) {
     return response => {
```

Comment:
duplicate - Can we extract csm logging into a new fn and use try-catch to be safe ?